IndustryData file populated

m0liere opened the pull request:
Wrote function in a new file to write the industry data to a new file sorting by state. I ran this function so the file "data/industries.json" is populated.\* Also another file i manually made was added so I could populate names as well as a "states" dictionary for each industryCode in  "data/industries.json"

*note on initial run, one or some of the senators in senators csv were not in our by-industry data set (i used the same function you duncan used in senatorContributions.py), i think you mentioned something about this before, just a heads up. Also I didn't record which senators were not present, let me know if you want me to do that

Commit:
completed population of IndustryData.json, added 2 new files one being the newly popualated industry donation data by state, and one being a json with industry names so they could be added to the data set... note: on initial run there were some senators in the csv that were not in our data set

Commit:
Merge branch 'master' of https://github.com/Duncan93/donationviz

Duncan93 commented:
This looks for the most part good. As to your question about missing senators, I need you to use the function I wrote that references the stateSenators.json file instead of the CSV file, as it is the one that contains the most up-to-date information that we are using in our maps, database, etc. Sorry if that was a little confusing; my code references both pretty frequently. This will probably require you to rerun your code, but read below before doing that.

We can't necessarily connect industries to organizations yet, but we can connect them to sectors and senators. Can you modify your code to create a JSON file that shows the industry contributions to each senator? It may take the form of keys being cids, and values being an array of industry contributions. Right now you have aggregate values, which is important for mapping purposes on the industry page, but it does not allow us to show industry impact for senators.

After you get this stuff done, I'll write some code to load your JSON files into the database so we can start working on final frontend matters.

Commit:
changed formatIndustryData file findSenators function so it references stateSenators.json

Commit:
Merge branch 'master' of https://github.com/Duncan93/donationviz

m0liere commented:
So I addressed the first issue, changed the senators being referenced to the ones in stateSenators.json.

To address the second question, right now senatorIndustryData is a dict with cid keys and an array of top industry contributions, did you want me to go through and make a file like that but with all industries as opposed to just the top? In that case I'd be calling "candIndByInd" (https://www.opensecrets.org/api/?method=candIndByInd&output=doc) for each industry for each senator.

Duncan93 commented:
If you can get all of the industry contributions, that would be great. It shouldn't require a significant modification to your existing code. It may turn out that we do only want the top 5-10 (assuming a senator even has that many), but I think it's better to err on the side of more data.
